docs/extensions/guides/renderer-extension.md

```diff
@@ -1 +1,425 @@
+
+### `globalPageMenus`
+
+Global page menus connect a global page to the cluster menu, which is the vertical strip along the left side of the Lens UI showing the available cluster icons, and the Add Cluster icon. Expanding on the example from [`globalPages`](#globalPages) a global page menu is added to the `HelpExtension` definition:
+
+``` typescript
+import { LensRendererExtension } from "@k8slens/extensions";
+import { HelpIcon, HelpPage } from "./page"
+import React from "react"
+
+export default class HelpExtension extends LensRendererExtension {
+  clusterPages = [
+    {
+      id: "help",
+      components: {
+        Page: () => <HelpPage extension={this}/>,
+      }
+    }
+  ];
+
+  globalPageMenus = [
+    {
+      target: { pageId: "help" },
+      title: "Help",
+      components: {
+        Icon: HelpIcon,
+      }
+    },
+  ];
+}
+```
+
+Global page menus are objects matching the `PageMenuRegistration` interface. They define the appearance of the global page menu item in the cluster menu and the behaviour when the global page menu item is activated (typically by a mouse click). The example above uses the `target` field to set the behaviour as a link to the global page with `id` of `"help"`. This is done by setting `target`'s `pageId` field to `"help"`. The global page menu item's appearance is defined by setting the `title` field to the text that is to be displayed as a tooltip in the cluster menu. The `components` field is used to set an icon that appears in the cluster menu. Thus when the `"Help"` icon is activated the contents of `ExamplePage` will be shown. This example requires the definition of another React-based component, `HelpIcon`, which has been added to `page.tsx`:
+
+``` typescript
+import { LensRendererExtension, Component } from "@k8slens/extensions";
+import React from "react"
+
+export function HelpIcon(props: Component.IconProps) {
+  return <Component.Icon {...props} material="help"/>
+}
+
+export class HelpPage extends React.Component<{ extension: LensRendererExtension }> {
+  render() {
+    return (
+      <div>
+        <p>Help</p>
+      </div>
+    )
+  }
+}
+```
+
+`HelpIcon` introduces one of Lens' built-in components available to extension developers, the `Component.Icon`. Built in are the [Material Design](https://material.io) icons. One can be selected by name via the `material` field. 
```

Comment:
Can link directly to the icon page? <https://material.io/resources/icons/>

Reply:
could, but the page has a link to them...